src/McBopomofo.cpp

```diff
@@ -368,7 +373,7 @@ McBopomofoEngine::McBopomofoEngine(fcitx::Instance* instance)
                       : _("Full Width Punctuation"),
               enabled ? _("Now using half width punctuation")
                       : _("Now using full width punctuation"),
-              1000);
+              kFcitx5NotificationTimeout);
```

Comment:
I cursorily inspected fcitx5's source code, and wonder if the value is used at all…

Reply:
Filed https://github.com/openvanilla/fcitx5-mcbopomofo/issues/117